docs/source/models/gbd2017/causes/tuberculosis/tuberculosis.rst

```diff
@@ -0,0 +1,135 @@
+.. _tuberculosis_model:
+
+=====================
+Modeling Tuberculosis
+=====================
+
+TB Overview
+-----------
+Tuberculosis (TB) is a disease caused by the bacteria Mycobacterium

```

Comment:
Are there relevant GBD sources we can cite here as well?  I think it's a good idea to link out to any info the GBD has published about their TB/Hiv estimates.